test/lex/backslash.f90

```diff
@@ -8,6 +8,10 @@
 ! RUN: %flang -c -fno-backslash %s
 ! RUN: not %flang -c -Mnobackslash %s 2>&1 | FileCheck %s
 ! RUN: not %flang -c -fbackslash %s 2>&1 | FileCheck %s
+! RUN: %flang -c -Mnobackslash -Mbackslash %s
+! RUN: %flang -c -fbackslash -fno-backslash %s
+! RUN: not %flang -c -Mnobackslash -Mbackslash -Mnobackslash %s 2>&1 | FileCheck %s
```

Comment:
Is the first instance of `-Mnobackslash` intentional? It doesn't match the pattern on other lines.

Reply:
yes. I intentionally added the combination of Mnobackslash -Mbackslash to check if compiler resolves to Mbackslash correctly.

Reply:
In that case, do you want to apply the same treatment to line 14?